src/Utilities/UtilityFunctions.h

```diff
@@ -70,6 +70,22 @@ inline void FairDivide(int ntot, int npart, IV& adist)
   adist[npart]=ntot;
 }
 
+/** Partition ntot over npart and the size of each partition is a multiple of base size
```

Comment:
I agree with you. An issue (#1090) has been created.
"change its guard constants so it doesn't look like it should be in the OHMMS directories" could you please explain it a bit more. I'm changing it as a template function to avoid potential multi definition issue and support size_t

Reply:
#ifndef OHMMS_UTILITYFUNCTIONS_H
#define OHMMS_UTILITYFUNCTIONS_H

Reply:
#ifndef OHMMS_UTILITYFUNCTIONS_H
#define OHMMS_UTILITYFUNCTIONS_H